README.md

```diff
@@ -84,6 +84,7 @@ The output hdf5 files each contain two datasets:
 | `prompt_keyword` | str | 'prompt' | 512 for gpt2 small, 1024 for gpt-xl, 2048 for gpt3-13B. | If your input json has a string keyword for prompt other than "prompt", place the keyword here. e.g Input_json: {"source": ... "target": ...} ->`prompt_keyword`='source'. |
 | `completion_keyword` | str | 'completion' | 512 for gpt2 small, 1024 for gpt-xl, 2048 for gpt3-13B. | If your input json has a string keyword for completion other than "completion", place the  keyword here. e.g Input_json: {"source": ... "target": ...} -> --completion_keyword='target'. |
 | `disable_space_separator` | bool | False | Include flag for True, no arguments |  If you include this flag, NO spaces will be prepended to the completion. (If you do not add this flag then a space is added to every completion if it does not already have a space). Including this flag is dangerous and not recommended because if you have input data like {"prompt": "hello." "completion": "how are you?"}, when the prompt and completion are combined it will look like "hello.how are you?" which will mess up the tokenization.--completion_keyword='target'. |
+| `keep_prompt_only_sequences` | bool | False | Include flag for True, no arguments | If you include this flag, packed sequences that contain no COMPLETION tokens will be dropped. The output jsonl file may contain such sequences depending on 1. if the input file contains prompt-only articles, and 2. if a sequence is too long, and packing results in a packed sequence only containing prompt tokens. This flag is off by default because training with prompt-only sequences with prompt_loss_weight=0.0 may lead to training instability. |
```

Comment:
If you include this flag, packed sequences with only prompt tokens will not be dropped. Data with only prompt will be dropped by default because training with prompt-only sequences with prompt_loss_weight=0.0 may lead to errors. Data is dropped because of one of the following conditions:  1. the input file data prompt completion pairs contains only a prompt. 2. If the sequence is truncated such that only prompt tokens remain